ibm/service/vpc/resource_ibm_is_instance.go

```diff
@@ -439,6 +439,11 @@ func ResourceIBMISInstance() *schema.Resource {
 							ValidateFunc: validate.InvokeValidator("ibm_is_instance", isInstanceBootAttachmentName),
 						},
 
+						isInstanceBootVolumeId: {
```

Comment:
Should this be a part of Instance from boot volume feature? and not resize boot volume?

Reply:
I get it, this is the generated boot volume id.